src/main/java/org/sagebionetworks/web/client/widget/header/Header.java

```diff
@@ -27,21 +30,33 @@ public Header(
     SynapseJSNIUtils synapseJSNIUtils,
     EventBus eventBus,
     CookieProvider cookies,
+    ClientCache localStorage,
     JSONObjectAdapter jsonObjectAdapter
   ) {
     this.view = view;
     this.cookies = cookies;
+    this.localStorage = localStorage;
     this.synapseJSNIUtils = synapseJSNIUtils;
     view.clear();
 
     view.setPresenter(this);
     initStagingAlert();
     view.getEventBinder().bindEventHandlers(this, eventBus);
-    if (cookies.getCookie(CookieKeys.COOKIES_ACCEPTED) == null) {
+    if (!localStorage.contains(AuthenticationControllerImpl.COOKIES_ACCEPTED)) {
       view.setCookieNotificationVisible(true);
     } else {
       view.setCookieNotificationVisible(false);
     }
+    if (
+      !localStorage.contains(
+        AuthenticationControllerImpl.NIH_NOTIFICATION_DISMISSED
+      )
+    ) {
+      view.setNIHAlertVisible(true);
+    } else {
+      view.setNIHAlertVisible(false);
+    }
```

Comment:
Opinionated code style, take it or leave it
```suggestion
    view.setCookieNotificationVisible(
      !localStorage.contains(AuthenticationControllerImpl.COOKIES_ACCEPTED)
    )
    view.setNIHAlertVisible(
      !localStorage.contains(AuthenticationControllerImpl.NIH_NOTIFICATION_DISMISSED)
    )
```

Reply:
Copied style from my more verbose days as a java developer :)